### PR TITLE
Reduced Off-World Off-License vendor prices

### PR DESCRIPTION
**Pull request by yclatious**
Because cracking a Code One :tm: with the boys shouldnt cost you 50 credits, cant have the good greyshits go broke buying beer cans.

Changed all normal beer cans to 15, didnt touch greyshit vodka at all.

🆑 
 * rscadd: Reduced prices on the Off World Off License vendor, enjoy lower prices for lower quality beer.

### CODE REVIEW

**Comment by MadmanMartian** [comment]
should have six packs of cans available via cargo.

The plastic sheething deals double damage to space carp if thrown at them, binding them for a while

**Comment by SonixApache** [comment]
a code one?

**Comment by FudgePucker** [comment]
>The plastic sheeting deals double damage  to space carp if thrown at them, binding them for a while.

Help keep space clean. Please recycle.

**Comment by yclatious** [comment]
@MadmanMartian WYCI

@SonixApache Beer name, pun on "Crack a cold one with the boys", only its "Crack a Code One wuth the boys"

**Comment by Blithering** [comment]
do the beer cans work in IED recipes? If so might want to up the price to 20 to keep it in line with other drinks

**Comment by yclatious** [comment]
Havent tested it, but I was honestly going to change the cans themselves to be 10 credits instead of 20 but for the sake of atomicity I didnt include it here, even if it is a somewhat meaningless change no doubt folks wouldnt care about. 

Besides, its unprofitable long term and a much bigger pain in the ass to actually do IEDs from these as they require a vending frame thats empty and cargo to order it, deliver it and then installing it on said empty vending frame, balance isnt disturbed by this as people who do IEDs want to rush and make them with the least effort possible, and the several soda machines already deliver on it, especialy with 13 Loko.

**Comment by yackemflam** [comment]
But beer is more expensive than sodas

**Comment by yackemflam** [comment]
What you should do is find the ratio between minimun wage/cost of drinks and compare it to the games wage/prices

**Comment by yackemflam** [comment]
Tldr beer should cost 30 credits

**Comment by yackemflam** [comment]
Or 150 credits for a 6 pack

**Comment by yackemflam** [comment]
Brb spriting 6packs

**Comment by yclatious** [comment]
Okay first off, learn to one post instead of five post, have some fucking decency you plebian, second off, like I said above Im going to change soda so it is cheaper than beer, its just basic atomicity to not bundle changes, thirdly, I am not making beer a 60% of a wage, it used to be a full wage, 50 credits, but thats just absurd, im making it less so by making it 30%, meaning 15 credits instead of 30, a wage should buy you least a couple beers, again, Im changing sodas if the PR is merged so it makes sense and sodas become cheaper than beer, till then, aint upping the price.

**Comment by FudgePucker** [comment]
Fun fact: malt liquor is ceaper than soda.
Down at the gas station two hueg cans costs three dollars.
